desc/objectives/_confinement.py

```diff
@@ -0,0 +1,198 @@
+"""Objectives for targeting energy confinement."""
+
+from desc.compute.utils import _compute as compute_fun
+from desc.compute.utils import get_profiles, get_transforms
+from desc.grid import QuadratureGrid
+from desc.utils import Timer, errorif
+
+from .normalization import compute_scaling_factors
+from .objective_funs import _Objective
+
+
+class HeatingPower(_Objective):
```

Comment:
I'd prefer something like `HeatingPowerISS04` to distinguish from other possible objectives for heating power we may eventually add